system-variables.md

```diff
@@ -1212,8 +1212,8 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
 - Scope: GLOBAL
 - Persists to cluster: Yes
 - Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): No
-- Default value: `128`. Before v7.6.0, the default value is `1`.
-- Range: `[1, 1024]`
+- Default value: `8192`. Before v7.6.0, the default value is `1`. Starting from v8.2.0, the default value is changed from `1024` to `8192`.
+- Range: `[1, 8192]`. Before v8.2.0, the value range is `[1, 1024]`.
```

Comment:
Please update the release notes for v8.2.0 to include changes (default value and value range) to the `tidb_auto_analyze_partition_batch_size` system variable:
https://github.com/pingcap/docs/blob/3b153e9fa9bf4f62fbeaff2bc60751817128dd18/releases/release-8.2.0.md?plain=1#L174-L183